tests/test_mask.py

```diff
@@ -0,0 +1,233 @@
+"""Tests for augur mask
+
+NOTE: Several functions are monkeypatched in these tests. If you change the arguments
+for any function in mask.py, check that it is correctly updated in this file.
+"""
+import argparse
+import os
+import pytest
+
+from Bio import SeqIO
+from Bio.Seq import Seq
+from Bio.SeqRecord import SeqRecord
+
+from augur import mask
+
+# Test inputs for the commands. Writing these here so tests are self-contained.
+@pytest.fixture
+def sequences():
+    return {
+        "SEQ1": SeqRecord(Seq("ATGC-ATGC-ATGC"), id="SEQ1"),
+        "SEQ2": SeqRecord(Seq("ATATATATATATATAT"), id="SEQ2")
+    }
+
+@pytest.fixture
+def fasta_file(tmpdir, sequences):
+    fasta_file = str(tmpdir / "test.fasta")
+    with open(fasta_file, "w") as fh:
+        SeqIO.write(sequences.values(), fh, "fasta")
+    return fasta_file
+
+TEST_VCF="""\
+##fileformat=VCFv4.2
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT
+SEQ	1	.	G	A	.		.
+SEQ	2	.	G	A	.		.
+SEQ	3	.	C	T	.		.
+SEQ	5	.	C	T	.		.
+SEQ	8	.	A	C	.		.
+"""
+
+@pytest.fixture
+def vcf_file(tmpdir):
+    vcf_file = str(tmpdir / "test.vcf")
+    with open(vcf_file, "w") as fh:
+        fh.write(TEST_VCF)
+    return vcf_file
+
+TEST_BED_SEQUENCE = [1,2,4,6,7,8,9,10]
+# IF YOU UPDATE ONE OF THESE, UPDATE THE OTHER.
+TEST_BED="""\
+Chrom	ChromStart	ChromEnd	locus tag	Comment	
```

Comment:
See the comment above for the `read_bed_file` function about not supporting an explicit header in BED files.